adapters/richaudience/richaudiencetest/supplemental/requestWrongRequest.json

```diff
@@ -0,0 +1,93 @@
+{
+    "mockBidRequest": {
+      "id": "4d3f84eb-787b-42fb-a0cf-062690dadce3",
+      "imp": [
+        {
+          "id": "div-gpt-ad-1460505748561-0",
+          "banner": {
+              "format" : [{
+                    "w": 300,
+                    "h": 250
+                  }]
+          },
+          "bidfloor": 1e-05,
+          "bidfloorcur": "USD",
+          "ext": {
+           "bidder": {
+              "pid": "OsNsyeF68q",
+              "supplyType": "site",
+              "test": false
+            }
+          }
+        }
+      ],
+      "site": {
+        "domain": "bridge.richmediastudio.com",
+        "page": "https://bridge.richmediastudio.com//ab083674fb8200b877a6983126e4477d/PrebidServer/indexRa.html?pbjs_debug=true"
+      },
+      "device": {
+        "ip": "11.222.33.44",
+        "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
+    },
+      "user": {
+        "buyeruid": "189f4055-78a3-46eb-b7fd-0915a1a43bd2a",
+        "ext": {}
+      }
+    },
+    
+    "httpCalls": [
+      {
+        "expectedRequest": {
+          "uri": "http://ortb.richaudience.com/ortb/?bidder=pbs",
+          "body": {
+            "id": "4d3f84eb-787b-42fb-a0cf-062690dadce3",
+            "imp": [
+              {
+                "id": "div-gpt-ad-1460505748561-0",
+                "tagid" : "OsNsyeF68q",
+                "secure": 1,
+                "banner": {
+                    "format" : [{
+                          "w": 300,
+                          "h": 250
+                        }]
+                },
+                "bidfloor": 1e-05,
+                "bidfloorcur": "USD",
+                "ext": {
+                  "bidder": {
+                    "pid": "OsNsyeF68q",
+                    "supplyType": "site",
+                   "test": false
+                  }
+                }
+              }
+            ],
+            "site": {
+              "domain": "bridge.richmediastudio.com",
+              "page": "https://bridge.richmediastudio.com//ab083674fb8200b877a6983126e4477d/PrebidServer/indexRa.html?pbjs_debug=true"
+            },
+            "device": {
+                "ip": "11.222.33.44",
+                "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
+            },
+            "user": {
+              "buyeruid": "189f4055-78a3-46eb-b7fd-0915a1a43bd2a",
+              "ext": {
+
+              }
+            }
+          }
+        },
+        "mockResponse": {
+          "status": 200
+        }
+      }
+    ],
+    "expectedMakeBidsErrors": [
+        {
+          "value": "unexpected end of JSON input",
+          "comparison": "literal"
```

Comment:
While some of the request parameters are different, this is testing the same condition as `pidNotORTB.json`.